tests/integration/block_createBlock_k8s_test.go

```diff
@@ -224,7 +230,7 @@ func (s *K8sBlockImageCreateSuite) isPVCBound(name string) bool {
 		if strings.TrimRight(status, "\n") == "'Bound'" {
 			return true
 		}
-		time.Sleep(time.Second * 1)
+		time.Sleep(time.Second * 3)
```

Comment:
I am leaving this alone because this is the only retry block in the package if we end up adding more retry blocks, I will add constants.